README.rst

```diff
@@ -225,6 +225,12 @@ response will contain an error object with a ``debug_url``:
 
 This tells you to find the traceback interpreter at ``<host>:<port>/debug/1234567890``.
 
+Logging
+-------
+
+The registry has a default logger, ``Registry._logger`` and it logs all errors that are not defined
+by ``typedjsonrpc``. You can configure the logger in ``Registry._get_default_logger()``.
```

Comment:
The logger is actually configurable in a slightly different way than this documentation. You should not have to change any source code in registry.py.